common/src/main/java/com/gregtechceu/gtceu/common/data/GTBlocks.java

```diff
@@ -615,22 +627,27 @@ private static BlockEntry<ActiveBlock> createFireboxCasing(BoilerFireboxType typ
             .addLayer(() -> RenderType::cutoutMipped)
             .tag(BlockTags.SAPLINGS)
             .item()
+            .model(GTBlocks::rubberTreeModel)
             .tag(ItemTags.SAPLINGS)
             .build()
             .register();
 
+    @ExpectPlatform
+    private static void rubberTreeModel(DataGenContext<Item, BlockItem> context, RegistrateItemModelProvider provider) {
+        throw new AssertionError();
+    }
+
```

Comment:
common/src/main/java/com/gregtechceu/gtceu/common/data/GTModels.java

move to this class.